src/Traversal/Sdk/Sdk.targets

```diff
@@ -45,7 +45,9 @@
     <ProjectReferenceTargets Remove="Publish" />
     <ProjectReferenceTargets Remove="Pack" />
 
+    <ProjectReferenceTargets Include="Build" Targets=".default" OuterBuild="true" />
     <ProjectReferenceTargets Include="Build" Targets=".default" />
```

Comment:
I actually wonder, should this be removed then? Or would that break non-multitargeting projects?

Reply:
I'm assuming the `OuterBuild="true"` is good enough?  Have you looked at how static graph API handles this attribute?

Reply:
`<ProjectReferenceTargets Include="Build" Targets=".default" />` would still be necessary to ensure the default targets are called on referenced non-multitargeting projects.
https://github.com/dotnet/msbuild/blob/main/documentation/specs/static-graph.md#multitargeting-details

In the case I'm addressing, the item wouldn't be necessary though.